releasenotes/notes/hamiltonian-vars-to-hz-fbe74c4fd4803d0e.yaml

```
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    In the `PulseBackendConfiguration` hamiltonian `vars` are now returned in Hz rather than GHZ.
```

I think the reno needs double ticks, right?
```suggestion
    In the ``PulseBackendConfiguration`` hamiltonian ``vars`` are now returned in Hz rather than GHz.
```

---

Resolved in 2e2f7c1.